development-kit/pkg/engines/leaks/rules.go

```diff
@@ -16,11 +16,56 @@
 package leaks
 
 import (
+	engine "github.com/ZupIT/horusec-engine"
 	"github.com/ZupIT/horusec-engine/text"
-	"github.com/ZupIT/horusec/development-kit/pkg/enums/engine/advisories/leaks/regular"
+	"github.com/ZupIT/horusec/development-kit/pkg/engines/leaks/regular"
+	"github.com/ZupIT/horusec/development-kit/pkg/utils/logger"
 )
 
-func AllRulesLeaksRegular() []text.TextRule {
+type Interface interface {
+	GetAllRules() (rules []engine.Rule)
+	GetTextUnitByRulesExt(projectPath string) ([]engine.Unit, error)
+}
+
+type Rules struct{}
+
+func NewRules() Interface {
+	return &Rules{}
+}
+
+func (r *Rules) GetAllRules() (rules []engine.Rule) {
+	rules = r.addLeaksRules(rules)
+	return rules
+}
+
+func (r *Rules) addLeaksRules(rules []engine.Rule) []engine.Rule {
+	for index := range allRulesLeaksRegular() {
+		rules = append(rules, allRulesLeaksRegular()[index])
```

Comment:
Change for usage item. See commentary in java/rules for more details

Reply:
Fixed